modules/ossm-federation-config-failover.adoc

```diff
@@ -0,0 +1,180 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+:_content-type: PROCEDURE ????????????????
+[id="ossm-federation-config-failover_{context}"]
+= Configuring a federated mesh for failover
+
+Failover is the ability to switch automatically and seamlessly to a reliable backup system, for example another server. In the case of a federated mesh, you can configure a service in one mesh to failover to a service in another mesh.
+
+You configure Federation for failover by setting the `importAsLocal` and `locality` settings in a `ServiceImportSet` resource and then configuring a `DestinationRule` that configures failover for the service to the locality specified in the `ServiceImportSet`.
+
+.Prerequisites
+
+* Two or more OpenShift Container Platform 4.6 or above clusters already networked and federated.
+* `ExportServiceSet` resources already created for each mesh peer in the federated mesh.
+* `ImportServiceSet` resources already created for each mesh peer in the federated mesh.
+* An account with the cluster-admin role.
+
+== Configuring an ImportServiceSet for failover
```

Comment:
I would put `==Configuring an ImportServiceSet for failover `into a new module and use that module as a procedure module. You cannot have multiple headings in a procedure file.